develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -0,0 +1,380 @@
+---
+title: Logs
+sidebar_order: 3
+---
+
+<Alert title="Note" level="warning">
+
+The Sentry Logs feature is under active development. The information in this document is subject to change.
+
+</Alert>
+
+This document defines the format used by Sentry to ingest logs, as well as the SDK API and behavior that create and send logs to Sentry.
+
+## Logs Protocol
+
+There are two ways to send logs to Sentry: via the `log` envelope and Sentry Log protocol (preferred), or the `otel_log` envelope and OpenTelemetry Log protocol.
+
+All SDKs are required to send logs via the `log` envelope and Sentry Log protocol. The `otel_log` envelope and OpenTelemetry Log protocol are just documented for completeness and to provide a reference for SDK authors. See the [Appendix A](#appendix-a-otel_log-envelope-item-payload) for the `otel_log` envelope item payload.
+
+### `log` Envelope Item Payload
+
+The `log` envelope item payload is a JSON object that represents a Sentry Log. Multiple `log` envelope items can be sent in a single envelope.
+
+```json
+{
+  "timestamp": 1544719860.0,
+  "trace_id": "5b8efff798038103d269b633813fc60c",
+  "level": "info",
+  "body": "User John has logged in!",
+  "attributes": {
+    "sentry.message.template": {
+      "string_value": "User %s has logged in!"
+    },
+    "sentry.message.parameters.0": {
+      "string_value": "John"
+    },
+    "sentry.environment": {
+      "string_value": "production"
+    },
+    "sentry.release": {
+      "string_value": "1.0.0"
+    },
+    "sentry.trace.parent_span_id": {
+      "string_value": "b0e6f15b45c36b12"
+    }
+  }
+}
+```
+
+It consists of the following fields:
+
+`timestamp`
+
+: **Number, required**. The timestamp of the log in seconds since the Unix epoch.
+
+`trace_id`
+
+: **String, required**. The trace id of the log.
+
+`level`
+
+: **String, required**. The severity level of the log. One of `trace`, `debug`, `info`, `warn`, `error`, `fatal` (in order of lowest to highest).
+
+`body`
+
+: **String, required**. The log body/message.
+
+`attributes`
+
+: **Object, optional**. A dictionary of key-value pairs of arbitrary data attached to the log. Attributes must also declare the type of the value. The following types are supported: `string`, `boolean`, `integer`, `double`.
+
+Note: `intValue` is a string because JSON only recognizes numbers as floating point values.
+
+Example:
+
+```json
+{
+  "attributes": {
+    "string_item": {
+      "stringValue": "value"
+    },
+    "integer_item": {
+      "intValue": "123"
+    },
+    "boolean_item": {
+      "boolValue": true
+    },
+    "double_item": {
+      "doubleValue": 123.456
+    }
+  }
+}
```

Comment:
I would like to propose a format as this, as it's more readable and future proof.
```
"attributes": {
    "sentry.release": {
         "value": "1.0.0",
         "type": "string",
    },
},
```